File: JCleanArchitectureApp/appkoin/src/main/java/com/woowrale/jcleanarchitecture/data/remote/ws/ApiService.java
package com.woowrale.jcleanarchitecture.data.remote.ws;

import com.woowrale.jcleanarchitecture.data.remote.model.ContactResponse;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import retrofit2.http.Url;

public interface ApiService {

    @GET
    Call<List<ContactResponse>> getContacts(@Url String apiContacts,
                                            @Query("source") String source,
                                            @Query("search") String query);

}

File: KCleanArchitectureApp/presentation/src/main/java/com/woowrale/kcleanarchitecture/data/remote/ws/ApiService.kt
package com.woowrale.kcleanarchitecture.data.remote.ws

import com.woowrale.kcleanarchitecture.data.remote.model.ContactResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query
import retrofit2.http.Url

interface ApiService {
    @GET
    fun getContacts(
        @Url apiContacts: String,
        @Query("source") source: String
    ): Call<List<ContactResponse>>
}
